### PR TITLE
/get computers

### LINKED ISSUE #49: Create Computer Table
This module should create an ``Computer`` table 

### Process
- Write a function that drops the ``Computer `` table if it already exists
- Write a function that creates the ``Computer `` table if it doesn't already exist
- Insert the appropriate data into the table
- Export the function

### Expected Result
When the function is called in ``build-db.js``, it should create a table in the database with the data from the json files.

### LINKED ISSUE #56: Create Computers MVC
### Context
This should create the model, route, and controller to fetch data from the Computers table.

### Associated Product Ticket
Issue #2 

### Process
1. Create a function that promises to GET all products
1. Create a function that promises to GET a single product
1. Create a function that promises to POST a product
1. Create a function that promises to PUT a product
1. Create a function that promises to DELETE a product

### Expected Result
This module should export the above functions

### PR DESCRIPTION
**Pull request by kenziebottoms**
# Description
Adds endpoints `GET /computers/` and `GET /computers/:id`

## Related Ticket(s)
fixes #56 

## Steps to Test Solution

1. comment out `routes/index.js:10`
1. comment out `routes/employees/index.js:6-8` and `10-11`
1. go to `http://localhost:8080/computers/3`
1. go to `http://localhost:8080/computers/`

### CODE REVIEW

File: controllers/employees/computersCtrl.js [hunk 1]
```diff
@@ -1,8 +1,29 @@
 "use strict";
 
-const appRoot = process.cwd();
+const computers = require("../../models/employees/ComputersModel");
 
-const model = require(appRoot + "/models/ComputersModel");
+module.exports.getComputerById = (req, res, next) => {
+  computers.getSingleComputer(req.params.id)
+    .then(data => {
+      if (data.length >= 1) {
```
Comment (TimAConner) [inline controllers/employees/computersCtrl.js:8]: Should I be doing this too?  I'm not checking if data is being returned (the length >= 1).  I'm only checking if there is an error.

**Review by TimAConner — APPROVED** [review]
Looks good!

File: controllers/employees/computersCtrl.js (hunk 1 above)
Comment (kenziebottoms) [reply controllers/employees/computersCtrl.js:8]: not 100% necessary either way, but it returns an error if there are no results instead of returning `[]`

**Review by DavidLarsKetch — APPROVED** [review]
:+1:

**Pull request merged**